.github/workflows/build.yml

```diff
@@ -0,0 +1,27 @@
+name: Build on PR
+
+on:
+  pull_request:
+    branches:
+      - '**'  # Runs on pull requests to any branch
+    types: [opened, reopened, synchronize]
+
+jobs:
+  build:
+    name: Build Application
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
+
+      - name: Install dependencies
+        run: npm install --frozen-lockfile
+      - name: Generate Prisma Client
+        run: npm run postinstall
```

Comment:
This is'nt needed as postinstall will run after npm install automatically and will generate prisma schema.

Reply:
step 25 can be skipped in this workflow

Reply:
Thanks for the suggestion done @Riya267